utilix/mongo_files.py

```diff
@@ -0,0 +1,574 @@
+"""
+Downloader and uploader for mongo files
+https://straxen.readthedocs.io/en/latest/config_storage.html
+https://github.com/XENONnT/straxen/blob/master/straxen/mongo_storage.py
+
+This module contains 5 classes:
+ - GridFsInterface: General Gridfs base class (not used)
+ - MongoDownloader: subclass of GridFsInterface for downloading files
+    using a MongoClient
+ - MongoUploader: subclass of GridFsInterface for uploading files
+    using a MongoClient
+ - DownloadAPI: subclass of MongoDownloader where all the MongoClient
+    interactions have been replaced by API calls.
+ - UploadAPI: subclass of MongoUploader where all the MongoClient
+    interactions have been replaced by API calls.
+"""
+
+import os
+import tempfile
+from datetime import datetime
+from warnings import warn
+import pytz
+import gridfs
+from tqdm import tqdm
+from shutil import move
+import hashlib
+from pymongo.collection import Collection as pymongo_collection
+import utilix

```

Comment:
@ershockley , sure I can try again but last time I tried, I got circular import issues if I remember well. You want to from `utilix import uconfig` to avoid doing work twice? Let me give it another go.